### ISSUE TITLE
Error when using plugin-template

### ISSUE THREAD

**Issue by jobongo** [eNMS-automation/eNMS#339]
Trying to create a plugin and had some issues getting the template-plugin to load. Get the following error:

`File "/usr/local/lib/python3.8/dist-packages/flask/blueprints.py", line 195, in __init__raise ValueError("'name' may not contain a dot '.' character.")`

Worked around this by either disabling that check in blueprints.py (did not want to do this) or renaming the name of the plugin in the  register_routes function:
```
    def register_routes(self, server, **kwargs):
        name = __name__.replace("eNMS.plugins.", "")
        blueprint = Blueprint(f"{name}_bp", name, **kwargs["blueprint"])
```
Since it gets the name of the module from the directory, I used replace so that I didn't have to change anything else.

Also need to change the plugin settings (settings.json) to include the full path:

```
  "blueprint": {
    "template_folder": "eNMS/plugins/plugin-template/templates",
    "static_folder": "eNMS/plugins/plugin-template/static",
    "static_url_path": "/template-static"
  },
```
Not sure if this had already been mentioned before but wasn't able to find it in release notes.

**Comment by afourmy** [comment]
flask recently released a new version (2.0) which isn't backward compatible, the solution for now is to stick to flask 1

**Comment by jobongo** [comment]
Gotcha. Didn't even check the version of flask I was running... Just did a clean install. Reverted back to 1.1.4 and everything is working fine. Thanks.
